scripts/apptester/apptester.py

```diff
@@ -164,8 +164,9 @@ def __init__(self, config_file, kaanode, kaauser, rootpath, testdir, sandboxfram
 
         # list of applications that must be skipped during building ant testing
         self.skipped = []
-
         self.applications = self._create_applications(config_file)
+        self.BINARY = 'BINARY'
```

Comment:
These properties are not logically connected with AppTesterFramework. Consider to move them to a separate class (for example AppFormat). The format is a property of the application.